docs/src/modules/brandingTheme.ts

```diff
@@ -456,16 +456,6 @@ export function getThemedComponents(theme: Theme): { components: Theme['componen
           },
         },
       },
-      MuiContainer: {
```

Comment:
This contradicts the default value we have for the container, that we have in the system.